aspnetcore/performance/rate-limit.md

```diff
@@ -140,7 +140,7 @@ The <xref:System.Threading.RateLimiting.PartitionedRateLimiter.CreateChained%2A>
 
 The following code uses `CreateChained`:
 
-:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRate2/Program.cs" id="snippet_3" highlight="21,51":::
+:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRate2/Program.cs" id="snippet_3" highlight="19,20,33":::
```

Comment:
I noticed this while being on the page, https://learn.microsoft.com/en-us/aspnet/core/performance/rate-limit?view=aspnetcore-9.0#create-chained-limiters.

BEFORE:
![image](https://github.com/user-attachments/assets/359dd857-299b-42ce-96a1-32afab85af3b)


AFTER:
![image](https://github.com/user-attachments/assets/e28c7e80-b871-4d3e-93ce-31c4eabe4dbd)